rules-tests/DowngradePhp80/Rector/Expression/DowngradeThrowExprRector/Fixture/throw_property_fetch.php.inc

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp80\Rector\Expression\DowngradeThrowExprRector\Fixture;
+
+use Symfony\Component\Console\Helper\ProgressBar;
+
+final class ThrowPropertyFetch
+{
+    public function __construct(
+        private ProgressBar|null $progressBar
+    ) {
+    }
+
+    public function run()
+    {
+        return $this->progressBar ?? throw new \InvalidArgumentException();
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\DowngradePhp80\Rector\Expression\DowngradeThrowExprRector\Fixture;
+
+use Symfony\Component\Console\Helper\ProgressBar;
+
+final class ThrowPropertyFetch
+{
+    public function __construct(
+        private ProgressBar|null $progressBar
+    ) {
+    }
+
+    public function run()
+    {
+        if ($this->progressBar === null) {
```

Comment:
it should change to isset instead for `??`

Reply:
Indeed. Could you update it?

Reply:
I will try.

Reply:
Fixed at https://github.com/rectorphp/rector-src/pull/1426